tests/test_addon_detail.py

```diff
@@ -442,3 +442,53 @@ def test_screenshot_keyboard_navigation(selenium, base_url, variables):
     assert '1' in addon.screenshots.screenshot_counter
     # send ESC to close the screenshot viewer
     addon.screenshots.esc_to_close_screenshot_viewer()
+
+
+@pytest.mark.nondestructive
+def test_add_to_collection_card(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # verifies that the Add to Collection card is present on the detail page
+    assert 'Add to collection' in addon.add_to_collection.collections_card_header
+    assert addon.add_to_collection.collections_select_field.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_release_notes(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert (
+        'Release notes for ' + addon.more_info.addon_version_number.text
```

Comment:
Could you use an f string here? Like how you do in the ```selenium.get()``` function?

Reply:
Done!